test/config/akamai-purger.json

```diff
@@ -1,7 +1,7 @@
 {
   "akamaiPurger": {
     "debugAddr": ":9666",
-    "purgeInterval": "1ms",
+    "purgeInterval": "800ms",
```

Comment:
I'm _pretty_ sure that this is going up because 1ms * 33 entries is too much. It would be good to document that.